circle.yml

```diff
@@ -1,6 +1,16 @@
 dependencies:
   cache_directories:
     - "~/nilearn_data"
+
+  pre:
+    # We need to remove conflicting texlive packages.
+    - sudo -E apt-get -yq remove texlive-binaries --purge
+    # Installing required packages for `make -C doc check command` to work.
+    - sudo -E apt-get -yq update
+    - sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install dvipng texlive-latex-base texlive-latex-extra
+    # Force another download for this data
+    - rm -rf /home/ubuntu/nilearn_data/adhd
```

Comment:
> Yes, it's a kludge. We need to fix the issue

This looks acceptable as a temporary solution, adding a more self-explanatory comment would be good though.